zerver/lib/onboarding.py

```diff
@@ -282,36 +282,107 @@ def send_initial_realm_messages(realm: Realm) -> None:
             initial_private_stream_name=Realm.INITIAL_PRIVATE_STREAM_NAME,
         )
 
-        content1_of_topic_demonstration_topic_name = (
-            _(
-                "This is a message on stream #**{default_notification_stream_name}** with the "
-                "topic `topic demonstration`."
+        content1_of_moving_messages_topic_name = remove_single_newlines(
```

Comment:
Small note is that I think `content1_of_moving_messages_topic_name` is a pretty weird set of preexisting variable names for these strings; feel free to rework that part of the system.

Reply:
For me, these variable names were good enough to easily understand what they mean.

> One theory would be to just not have these variables at all, and just have the remote_single_newlines calls inline below, maybe like this:

This actually decreases code readability. The contents are ~8 (one message is 15) lines, so including them inline with .format() doesn't look good to me.

I think, we should just try to improve the variable names?

Reply:
Sounds good to me, and can be a follow-up.